netbout-web/src/main/java/com/netbout/rest/bout/TkInvite.java

```diff
@@ -46,10 +57,23 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 2.14
+ * @checkstyle ClassDataAbstractionCouplingCheck (210 lines)
  */
 final class TkInvite implements Take {
 
     /**
+     * Size of buffer.
+     */
+    private static final int BUFFER_SIZE = 8192;
+
+    /**
+     * RegEx to validate mail.
+     */
+    private static final Pattern MAIL_MASK = Pattern.compile(
+        "[a-z_\\.\\-A-Z0-9]+[@][a-z_\\.\\-A-Z0-9]+"
```

Comment:
@krzyk I'll improve that regex but I don't agree with the first regex on that site. It was created a lot of new TLD's that will be ignored using that regex. Any TLD that uses more than 4 chars, like `.travel`, `.airforce`, `.archi` and a lot of other TLD's will be ignored.